src/cpu/x64/jit_brgemm_conv.hpp

```diff
@@ -51,31 +51,8 @@ struct brgemm_convolution_fwd_t : public primitive_t {
 
         ~pd_t() = default;
 
-        // ------- DECLARE_COMMON_PD_t -----
-        pd_t *clone() const override {
-            auto new_pd = utils::make_unique<pd_t>(*this);
-            if (!new_pd->is_initialized()) return nullptr;
-            new_pd->brgs_.resize(brgs_sz_);
-            for (int i = 0; i < brgs_sz_; i++) {
-                new_pd->brgs_[i] = brgs_[i];
-                new_pd->bd_masks[i] = bd_masks[i];
-            }
```

Comment:
Seems like `DECLARE_COMMON_PD_T` doesn't contain these lines. Are you sure these change is safe in terms of node cloning procedure?

Reply:
@xipingyan  Thank you for clarification! LGTM now.